roadmap/roadmap.md

```diff
@@ -0,0 +1,90 @@
+# The Astropy Roadmap
+
+- [Status Legend](#Status-Legend)
+- [Functionality](#Functionality)
+- [Hardware and Performance](#Hardware-and-Performance)
+- [Learn and User Support](#Learn-and-User-Support)
+- [Community Building and Sustainability](#Community-Building-and-Sustainability)
+- [Infrastructure, Documentation](#Infrastructure-Documentation)
+- [Government, Management, and Personnel](#government-management-and-personnel)
+
+
+## Status Legend
+
+### Green :green_circle:
+- Group or person leading effort is identified and sufficiently supported.
+- Effort is already underway and/or expected to be implemented in the “near term.”
+
+### Orange :large_orange_diamond:    
+- Item is already defined (e.g., issue, APE exists), but not sufficiently underway.
+- Needs more people and/or resources to make it happen. 
+  - *For example, funding applied for, but not awarded.*
+  
+### Red :red_square:  
+- Consensus of the community is that the item is a priority.
+- Item won’t happen without more people/resources.
+  - *For example, there’s no one available to write a funding proposal or lead recruitment efforts.*
+  
+## Roadmap 2021 Project Board
+
+Issues relevant to the Roadmap goals can be added and organized on the [Astropy Roadmap 2021 Project Board](https://github.com/orgs/astropy/projects/7). The goal of the project board is to give a current overview of the status of our progress and provide access to the details of the work associated with roadmap items. It will be updated continuously.
+
+## Functionality
+
+- :large_orange_diamond: Provide next-generation spectroscopic reduction, analysis, and visualization tools usable by individual researchers and larger surveys.

```

Comment:
As a non-specialist in the field, “next-generation” seems a bit fuzzy. How do next-generation tools differ from current-generation tools?

Reply:
Personal opinion: Current generation is more like PyRAF scripts, home-grown code that is very specialized to a particular researcher; while next-generation is like Star Trek utopia that allows interoperability and reproducible science.